### PR TITLE
E2E `BellatrixE2EForkEpoch` constant

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

E2E

**What does this PR do? Why is it needed?**

Adds `BellatrixE2EForkEpoch` constant to E2E tests.

**Which issues(s) does this PR fix?**

Part of #10228

### CODE REVIEW

**Review by terencechain — COMMENTED** [review]
Have we thought about shortening the altair/bellatrix epochs to 4 and 8? or even 3 and 6? I wonder what trade-offs are at play here

**Comment by rkapka** [comment]
Simply changing the value of the Altair constant results in errors, so it will require a bit of work if we want to reduce the amount of epochs.

**Review by nisdas — COMMENTED** [review]
You need to add a corresponding value in the params file for the bellatrix fork epoch. The value of 10 will fail as that is when our e2e ends. So for this case, a value of 7/8 should be fine.

File: config/params/testnet_e2e_config.go [hunk 1]
```diff
@@ -42,8 +45,12 @@ func E2ETestConfig() *BeaconChainConfig {
 	e2eConfig.DepositChainID = 1337   // Chain ID of eth1 dev net.
 	e2eConfig.DepositNetworkID = 1337 // Network ID of eth1 dev net.
 
-	// Altair Fork Parameters.
+	// Fork Parameters.
 	e2eConfig.AltairForkEpoch = altairE2EForkEpoch
+	e2eConfig.BellatrixForkEpoch = bellatrixE2EForkEpoch
```
Comment (nisdas) [inline config/params/testnet_e2e_config.go:50]: You need to revert this for now, as this will cause prysm to move to the merge fork during its e2e run.

**Review by nisdas — APPROVED** [review]

**Pull request merged**